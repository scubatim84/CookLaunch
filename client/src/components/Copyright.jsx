import React from 'react';
import { Link, Typography } from '@material-ui/core';

const Copyright = () => {
  return (
    <Typography variant='body2' color='textSecondary'>
      {'Copyright Â© '}
      <Link color='inherit' href='http://www.cooklaunch.com'>
        Cook Launch
      </Link>{' '}
      {new Date().getFullYear()}
      {'.'}
    </Typography>
  );
};

export default Copyright;
