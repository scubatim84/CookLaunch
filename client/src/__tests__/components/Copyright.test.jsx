import React from 'react';
import {shallow} from 'enzyme';
import Copyright from '../../components/Copyright';

describe('Copyright', () => {
  it('Renders component without crashing', () => {
    shallow(<Copyright />);
  });
});
