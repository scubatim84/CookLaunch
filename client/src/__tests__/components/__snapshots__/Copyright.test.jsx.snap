import React from 'react';
import {shallow} from 'enzyme';
import Copyright from '../../components/Copyright';

describe('Copyright', () => {
  it('Renders correctly', () => {
    const wrapper = shallow(<Copyright />);

    expect(wrapper).toMatchSnapshot();
  });
});
